runtime/lua/vim/filetype.lua

```diff
@@ -1550,7 +1560,7 @@ function M.add(filetypes)
   end
 
   for k, v in pairs(filetypes.pattern or {}) do
-    pattern[normalize_path(k)] = v
+    pattern[normalize_path(k, 1)] = v
```

Comment:
Minor nit, otherwise LGTM
```suggestion
    pattern[normalize_path(k, true)] = v
```

Reply:
Please excuse my Lua newbness. 🙇🏻